spreadsheet/formula/horizontalrange.go

```diff
@@ -56,3 +56,13 @@ func cellRefsFromHorizontalRange(ctx Context, rowFrom, rowTo int) (string, strin
 	to := lastColumn + strconv.Itoa(rowTo)
 	return from, to
 }
+
+// ToString returns a string representation of a horizontal range.
+func (r HorizontalRange) ToString() string {
+	return r.horizontalRangeReference()
+}
+
+// MoveLeft returns the same object as moving ranges to left does not affect horizontal ranges.
```

Comment:
Perhaps if moving left, it should not implement the MovableExpression or Movable interface?
Then we can only call the MoveLeft if it fulfills the interface

Reply:
MoveLeft is required for these cases too, it just should return Expression unchanged. Say, for this case =SUM(1:4) is not changed when columns are removed but it should be returned as 1:4 as well